Hygeine

# Motivation
The samples are intended to give users a good starting place on how to do particular use cases w/ SCDF. We need to make sure what we are recommending is up-to-speed w/ latest libs and usage of the libs. We should also make sure the samples we recommend are still relevant.

# Steps
- [ ] **STEP1**: inventory the samples we should support (delete the irrelevant/unwanted ones)
- [ ] **STEP2**: update all libs to the latest supported versions
- [ ] **STEP3**: update all usage of the APIS in the libs to the latest supported versions

# Step Details 

## STEP1
- I will need input from others on what we should/not support (@cppwfs @jvalkeal )

## STEP2
- **TODO**: list version of major libs to support here (assuming latest supported version of SCDF deps)

## STEP3
-  Replace deprecations and any "glaring" older API uses; does not have to be exhaustive.

```java
INFO] Reactor Summary for ticktock 1.0.0-SNAPSHOT:
[INFO] 
[INFO] multi-output-time-rabbit ........................... SUCCESS [ 26.027 s]
[INFO] multi-input-log-rabbit ............................. SUCCESS [  4.110 s]
[INFO] multi-output-time-kafka ............................ FAILURE [  1.696 s]
[INFO] multi-input-log-kafka .............................. SKIPPED
[INFO] ticktock ........................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  34.873 s
[INFO] Finished at: 2022-01-16T16:00:32Z
[INFO] ------------------------------------------------------------------------
Error:  Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.2.0:jar (javadoc) on project multi-output-time-kafka: MavenReportException: Error while generating Javadoc: 
Error:  Exit code: 1 - javadoc: warning - No source files for package org.springframework.cloud.stream.app.time.source.kafka
Error:  javadoc: error - No public or protected classes found to document.
Error:  
Error:  Command line was: /usr/lib/jvm/adoptopenjdk-8-hotspot-amd64/jre/../bin/javadoc @options @packages
Error:  
Error:  Refer to the generated Javadoc files in '/home/runner/work/spring-cloud-dataflow-samples/spring-cloud-dataflow-samples/multi-io-samples/ticktock/multi-output-time-kafka/target/apidocs' dir.
Error:  -> [Help 1]
Error:  
Error:  To see the full stack trace of the errors, re-run Maven with the -e switch.
Error:  Re-run Maven using the -X switch to enable full debug logging.
Error:  
Error:  For more information about the errors and possible solutions, please read the following articles:
Error:  [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
Error:  
Error:  After correcting the problems, you can resume the build with the command
Error:    mvn <args> -rf :multi-output-time-kafka
Error: Process completed with exit code 1.
```

From multi-io-tick-tock...
```java
[INFO] ------------------------------------------------------------------------
Error:  Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.2.0:jar (javadoc) on project multi-output-time-kafka: MavenReportException: Error while generating Javadoc: 
Error:  Exit code: 1 - javadoc: warning - No source files for package org.springframework.cloud.stream.app.time.source.kafka
Error:  javadoc: error - No public or protected classes found to document.
Error:  
Error:  Command line was: /usr/lib/jvm/adoptopenjdk-8-hotspot-amd64/jre/../bin/javadoc @options @packages
Error:  
Error:  Refer to the generated Javadoc files in '/home/runner/work/spring-cloud-dataflow-samples/spring-cloud-dataflow-samples/multi-io-samples/ticktock/multi-output-time-kafka/target/apidocs' dir.
Error:  -> [Help 1]
Error:  
Error:  To see the full stack trace of the errors, re-run Maven with the -e switch.
Error:  Re-run Maven using the -X switch to enable full debug logging.
Error:  
Error:  For more information about the errors and possible solutions, please read the following articles:
Error:  [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
Error:  
Error:  After correcting the problems, you can resume the build with the command
Error:    mvn <args> -rf :multi-output-time-kafka
Error: Process completed with exit code 1.
```

--- a/dataflow-website/stream-developer-guides/streams/standalone-stream-sample/pom.xml
+++ b/dataflow-website/stream-developer-guides/streams/standalone-stream-sample/pom.xml
@@ -42,7 +42,7 @@
                 <artifactId>spring-boot-maven-plugin</artifactId>
                 <configuration>
                     <image>
-                        <name>springcloudstream/multi-io-sample-${project.artifactId}:${project.version}</name>
+                        <name>springcloudstream/${project.artifactId}:${project.version}</name>
                     </image>
                 </configuration>
             </plugin>

--- a/multi-io-samples/kafka-streams/user-clicks-per-region-1.2.x/user-clicks-per-region-processor/pom.xml
+++ b/multi-io-samples/kafka-streams/user-clicks-per-region-1.2.x/user-clicks-per-region-processor/pom.xml
@@ -134,7 +134,7 @@
 						<name>springcloudstream/multi-io-sample-${project.artifactId}:${project.version}</name>
 						<env>
 							<BP_IMAGE_LABELS>
-								configuration-properties.inbound-ports="${configuration-properties.inbound-ports}",configuration-properties.outbound-ports="${configuration-properties.outbound-ports}"
+								configuration-properties.inbound-ports="${configuration-properties.inbound-ports}" configuration-properties.outbound-ports="${configuration-properties.outbound-ports}"
 							</BP_IMAGE_LABELS>
 						</env>
 					</image>

